docs/reference/remap/to_syslog_facility.cue

```diff
@@ -0,0 +1,40 @@
+package metadata
+
+remap: functions: to_syslog_facility: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The facility code."
+			required:    true
+			type: ["integer"]
+		},
+	]
+	return: ["string"]
+	category:    "Coerce"
+	description: """
+		Converts a Syslog [facility code](\(urls.syslog_facility)) into its corresponding keyword,
+		i.e. 0 into `"kern"`, 1 into `"user", etc.
+		"""
+	examples: [
+		{
+			title: "Success"
+			input: {
+				SYSLOG_FACILITY: "4"
```

Comment:
Nit: all of our other examples use fields with lowercase names.

Reply:
adjusted case for consistency